msupply-foundation/open-msupply
# 5398 use store pref to hide extra columns

Fixes #5398

# 👩🏻‍💻 What does this PR do?
Use store pref to hide extra columns in program requisitions until global pref is implemented

![Screenshot 2024-11-14 at 4 54 02 PM](https://github.com/user-attachments/assets/83ae2fa2-4319-466f-8136-f4d3be5a047c)

# 🧪 Testing



1. Enable preference
- [ ] Have programs set up
- [ ] Go to your store -> Preferences
- [ ] Enable the `Show extra fields on requisition` preference
- [ ] Go to OMS -> Sync
- [ ] Create a program requisition for any store
- [ ] See all the extra columns
- [ ] Click on a line, also see the extra columns in the line edit page

2. Disable Preference
- [ ] Go back to mSupply
- [ ] Go to your store -> Preferences
- [ ] Disabled `Show extra fields on requisition` preference
- [ ] Go to OMS -> Sync
- [ ] Create program requisition for any store
- [ ] See all extra columns gone
- [ ] Click on lines, extra columns should be gone from here too

ALTERNATIVELY
Just set up two stores on your OMS site, have one with pref and one without and follow above

# 📃 Documentation

  1.
  2.
